### PR TITLE
Update code coverage to 100%

### PR DESCRIPTION
**Pull request by stramel**
**Summary:**

- Updates code coverage by ignoring a couple of lines that are immediately replaced  and removing duplicated logic
- Swap a couple ternary operators for boolean logic
- Swap a couple `{ [key: number]: boolean}` for `Record<number, boolean>`

### CODE REVIEW

**Comment by stramel** [comment]
@Buuntu You may need to add a Personal Access Token called `GITHUB_TOKEN` that can comment so the size check works.

```
Size Differences:
                                  dist/index.js ⏤  152 B
      dist/react-final-table.cjs.development.js ⏤  3.03 kB (+9 B)
   dist/react-final-table.cjs.production.min.js ⏤  1.9 kB (-5 B)
                  dist/react-final-table.esm.js ⏤  2.97 kB (+11 B)
```

File: src/hooks.tsx [hunk 1]
```diff
@@ -278,8 +277,8 @@ export const useTable = <T extends DataType>(
       perPage: 10,
       canNext: true,
       canPrev: false,
-      nextPage: () => {},
-      prevPage: () => {},
+      nextPage: /* istanbul ignore next */ () => {},
```
Comment (Buuntu) [inline src/hooks.tsx:280]: Not sure how I feel about having these comments everywhere but I'm okay with it for now.

**Review by Buuntu — APPROVED** [review]
I think GITHUB_TOKEN is automatically generated: https://docs.github.com/en/actions/reference/authentication-in-a-workflow

**Comment by stramel** [comment]
Right, there is a default `GITHUB_TOKEN` but I don't believe that one has permission to comment on PRs. Most people replace it with a Personal Access Token.

**Comment by Buuntu** [comment]
> > I think GITHUB_TOKEN is automatically generated: https://docs.github.com/en/actions/reference/authentication-in-a-workflow
> 
> Right, there is a default `GITHUB_TOKEN` but I don't believe that one has permission to comment on PRs. Most people replace it with a Personal Access Token.

It wouldn't even let me add a GITHUB_TOKEN secret when I tried.  I guess it's reserved for the automatic one.

**Comment by stramel** [comment]
I believe I have used `GH_TOKEN` and `PAT` before for the secret name.

**Comment by Buuntu** [comment]
I added a secret called `GH_ACCESS_TOKEN` with full repo access.  Give it a try.

**Comment by stramel** [comment]
> I added a secret called GH_ACCESS_TOKEN with full repo access if you want to change it to that and try it real quick. I think it might have more to do with it being a fork though?

Appears that it is due to the fork :/

**Comment by Buuntu** [comment]
> > I added a secret called GH_ACCESS_TOKEN with full repo access if you want to change it to that and try it real quick. I think it might have more to do with it being a fork though?
> 
> Appears that it is due to the fork :/

So do you want to remove it from GH actions or is there a solution for forks?

**Comment by stramel** [comment]
> So do you want to remove it from GH actions or is there a solution for forks?

I can remove it in another PR and look for a better overall solution. Being able to test size locally is useful too. So something like `size-limit` might be good. I think we can continue with this PR

**Pull request merged**